Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN go mod download
 COPY . .
 ARG version
 ENV version=${version}
-RUN ./scripts/build-engine.sh
+RUN go build -o engine -ldflags="-X 'github.com/mesg-foundation/engine/version.Version=$version'" core/main.go
```

Comment:
can't we do a `make install` or something similar

Reply:
I intentionally haven't used any make commands inside a Dockerfile or cricleci. I wanted to see how it will envolve and make that transition later. The first step is to move everything there so we can use it easily.